django/db/backends/base/creation.py

```diff
@@ -268,6 +273,17 @@ def destroy_test_db(self, old_database_name=None, verbosity=1, keepdb=False, num
             settings.DATABASES[self.connection.alias]["NAME"] = old_database_name
             self.connection.settings_dict["NAME"] = old_database_name
 
+    def _restore_initial_data_migrations(self):
+        # First, clean the database
+        call_command('flush', verbosity=0, interactive=False,
```

Comment:
It was more for "security" to be 100% sure about database state, in case I have missed some possibilities.

But as `SimpleTestCase` doesn't allow database queries, `TestCase` rollbacks everything and `TransactionTestCase` flush everything at the end of the test, I guess it's useless...

Do you think this approach makes more sense and should I create some related tests ?